### PR TITLE
Onesignal create push 0.0.1

### PR DESCRIPTION
**Pull request by sonifex**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.
For steps added for the first time, the GitHub option **Allow edits from maintainers** is required (See the documentation on how to select it: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by sonifex** [comment]
Hi,
I cant see the reason for build failure. Can you give me more detail? Then I can update the pr.

File: steps/onesignal-create-push/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+title: |
+  OneSignal Create Push
```
Comment (lpusok) [inline steps/onesignal-create-push/0.0.1/step.yml:2]: I reccomend naming the step `Send OneSignal push message` or similar, based on https://devcenter.bitrise.io/contributors/create-your-own-step/#the-title.

File: steps/onesignal-create-push/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- macos
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+type_tags:
+- notification
+toolkit: {}
+deps:
```
Comment (lpusok) [inline steps/onesignal-create-push/0.0.1/step.yml:28]: Please remove deps, in case git and wget are unused.

**Comment by lpusok** [comment]
@sonifex Created a PR to allow validation to pass, please merge: https://github.com/sonifex/bitrise-steplib/pull/1

**Comment by sonifex** [comment]
I have created a new pr #2491

**Pull request merged**